docs/adguard-browser-extension/mv3-version.md

```diff
@@ -0,0 +1,78 @@
+---
+title: MV3 version
+sidebar_position: 5
+---
+
+As MV2 extensions are being phased out in the Chrome Web Store, Chrome users may need to switch to MV3 extensions that are compatible with Chrome’s new API Manifest V3. Our thoughts on this topic are extensively discussed on [our blog](https://adguard.com/en/blog/tag/manifest-v3.html).
+
+Our MV3 extension effectively blocks ads and trackers while seamlessly managing social widgets, banners, and video ads. Most users will not notice any operational differences, but there are some limitations and changes to be aware of.
+
+## Where to find our extensions
+
+The **MV3 version** has replaced our old extension in the [Chrome Web Store](https://chromewebstore.google.com/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg).
+
+The **beta version of MV3** is still available in the [Chrome Web Store](https://chromewebstore.google.com/detail/adguard-adblocker-mv3-exp/apjcbfpjihpedihablmalmbbhjpklbdf).
+
+The old beta extension has been renamed to [**AdGuard Ad Blocker MV2**](https://chromewebstore.google.com/detail/adguard-adblocker-beta/gfggjaccafhcbfogfkogggoepomehbjl) and will be supported until Google phases it out.
+
+## Main changes in features and setting
+
+- **No auto and manual filter updates.** The options *Auto-update filters* and *Check filters update* are no longer available in the *Filters* tab. Since some of the rules are now applied in DNR form, we can’t update filters on request, only through the full process of updating the extension along with the review in the stores.
+
+  > In the future, we want to implement differential filter updates, similar to our MV2 extension. When we add this, the manual and automatic update options will return.
+
+- **Service worker functionality.** Chrome has implemented a workaround so that service worker doesn’t go to sleep. Why is this important?
+
+   When the service worker is inactive, it affects the way rules are applied, with a few seconds of delay and glitches. The Chrome workaround helps with this problem, but it is not a foolproof solution. Chrome can always remove the workaround and the glitching will return. We are working on our own solution to reduce the glitching delay to a minimum, but it will still be more noticeable and slower than it was in MV2.
+
+- **Filtering log**
+
+  ![Filtering log *border](https://cdn.adtidy.org/content/blog/new/log.png)
+
+  Due to DNR restrictions, we can’t show exactly which rule worked, but we will provide an “approximate rule that was triggered” based on our engine. For precise information, you’ll need to install the “unpacked” form of the extension in your browser yourself. That is, you need to clone [our repository](https://github.com/AdguardTeam/AdguardBrowserExtension/tree/v5.0), “build” the extension, switch the browser to Developer mode, and only in this case you will be able to use the tools for debugging filters.
```

Comment:
The whole unpacking mechanim should be a separate article in the knowledge base (under solving problems probably) that we can link from inside the extension. 

This article should be written by the devs and it should explain in detail how to get an unpacked extension and, more importantly, how to modify static filters.

Reply:
added a separate article